csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.csproj

```diff
@@ -6,7 +6,7 @@
 
   <PropertyGroup>
     <IncludeMobileTargets>true</IncludeMobileTargets>
-    <BaseTargets>netstandard2.0</BaseTargets>
+    <BaseTargets>netstandard2.0;netcoreapp3.1;net6.0</BaseTargets>
```

Comment:
We probably should not have netcoreapp3.1 here.

Reply:
I retested it again.
It could work without netcoreapp3.1 now so it's removed
![image](https://github.com/microsoft/onnxruntime/assets/16190118/cab3b9b2-2e64-41f9-acd7-6730d1af8dac)